fhir-model/src/main/java/com/ibm/watson/health/fhir/model/util/FHIRUtil.java

```diff
@@ -835,4 +836,15 @@ public static boolean isFailure(IssueSeverity severity) {
             return true;
         }
     }
+
+    public static <K, V> Map<K, V> createLRUCache(int maxEntries) {
+        return Collections.synchronizedMap(new LinkedHashMap<K, V>(maxEntries, 0.75f, true) {
```

Comment:
is it worth having a dedicated class for the anonymous inner class that extends LinkedHashMap ?

Reply:
Perhaps if it gets more widely used. Right now it’s only used in 2-3 places in FHIREvaluator.